sc_server_auth/create_admin.py

```diff
@@ -0,0 +1,53 @@
+from argparse import ArgumentParser
```

Comment:
Who can add admin now? Do I understand correctly what this script can be executed only locally on the server?

Reply:
Yes, like manage.py in django